index.js

```diff
@@ -213,10 +237,14 @@ const createFocusTrap = function (elements, userOptions) {
   const checkFocusIn = function (e) {
     // In Firefox when you Tab out of an iframe the Document is briefly focused.
     if (containersContain(e.target) || e.target instanceof Document) {
-      return;
+      if (containersContain(e.target)) {
```

Comment:
Does it feel redundant to check this twice?  `containersContain(e.target)` in the line above and `containersContain(e.target)` here again?

Reply:
Yes, very! 😄  Thanks for pointing that out, fixed.